python/src/aiconfig/schema.py

```diff
@@ -59,9 +59,23 @@ class PromptMetadata(BaseModel):
     class Config:
         extra = "allow"
 
+class Attachment(BaseModel):
+    '''
+    Attachment used to pass data in PromptInput for non-text inputs (ex: image, audio)
+    '''
+    # The data representing the attachment
+    data: Any
+    # The MIME type of the result. If not specified, the MIME type will be assumed to be plain text.
+    mime_type: Optional[str] = None
```

Comment:
Left the same comment in the GPT4-V PR as well: do we need two "mime type" specific fields here -- one for the serialized data representation and one for what the data actually is? For example, if we have an image attachment the mime_type could be img/png but the serialized data itself could be uri, base64 string, etc.

We need a way to understand both:
- what the data is for (in order to show the UI for attaching image input / showing after attachment)
- how the data is saved (in order to render the img to show it)